pages/pipelines/permissions.md.erb

```diff
@@ -114,3 +114,36 @@ From the Member Permissions page, organization admins can toggle whether or not
 - Disconnect agents
 
 If your organization has teams enabled, the pipeline creation permissions are managed at a team level. Pipeline creation permission controls can be found on the Teams Settings page. Without teams enabled, the pipeline creation permission control can be found on the Member Permissions page. 
+
+Organization admin can delete orgranization members. To do it, the admin needs to:
+  1. Find the ID for the user to be deleted ("Jane Doe"):  
+```bash
```

Comment:
Probs needs a blank line, and the code isn't bash. `graphql` should work nicely.